src/page/stableMatching/Input.jsx

```diff
@@ -764,6 +789,13 @@ export default function InputPage() {
         </div>
 
       </div>
+      <div className="d-flex align-items-center">
```

Comment:
<div className="d-flex align-items-center">
        <div className="me-2">
          Use parallel RBO driver (for development purpose):
        </div>
        <Checkbox initialChecked={isUseParallelDriver}
                  onChange={handleChangeIsUseDriver}/>
      </div>
      
      
      Bỏ cả cục này đi nhé